tutorials/Ocean/eady_instability.jl

```diff
@@ -0,0 +1,331 @@
+# # Eady instability
+#
+# This script simulates the instability of a sheared, free-surface
+# flow using `ClimateMachine.Ocean.HydrostaticBoussinesqSuperModel`.
+
+using Printf
+
+ENV["GKSwstype"] = "nul"
```

Comment:
```suggestion
```
I think this is already set when the docs are run

Reply:
True --- but it's not set when we run examples locally. I can still remove it but then it has to be added in to manually run examples on headless systems, which is annoying. Please advise.